### PR TITLE
CR_1.18 | status is changed on pending

### PR DESCRIPTION
**Pull request by SvetlanaKanel**
https://trello.com/c/aO2lYEeq/1216-atcr118-verify-that-after-clicking-on-the-button-confirm-ticket-status-is-changed-on-pending

### CODE REVIEW

File: cypress/e2e/testBookingReservation/Reservation/CR_1.cy.js [hunk 1]
```diff
@@ -101,6 +101,14 @@ describe('Popup window parameters verification after the reservation was complet
     it("CB_1.17 | Verify Button Message to operator is exists", function() {
         bookingPopup.getBookingBtnMessageToOperator().should('include.text', this.bookingPopUpData.messageToOperator)
     });   
+
+    it('CR_1.18 | Verify that after clicking on the button "Confirm ticket" status is chenged on pending', function() {
+        bookingPopup.getBookingStatus().should('have.text', 'Reserved');
```
Comment (Rom-T) [inline cypress/e2e/testBookingReservation/Reservation/CR_1.cy.js:106]: Please move data in the fixtures

File: cypress/e2e/testBookingReservation/Reservation/CR_1.cy.js [hunk 2]
```diff
@@ -101,6 +101,14 @@ describe('Popup window parameters verification after the reservation was complet
...
+
+        bookingPopup.getConfirmTicketButton().click();
+
+        bookingPopup.getBookingStatus().should('have.text', 'Pending');
```
Comment (Rom-T) [inline cypress/e2e/testBookingReservation/Reservation/CR_1.cy.js:110]: Please move data in the fixtures

File: cypress/e2e/testBookingReservation/Reservation/CR_1.cy.js (hunk 1 above)
Comment (SvetlanaKanel) [reply cypress/e2e/testBookingReservation/Reservation/CR_1.cy.js:106]: точно!

**Review by Rom-T — APPROVED** [review]
Great job!

**Pull request merged**